### ISSUE TITLE
miou0.4 ,kappa是负值

### ISSUE THREAD

**Issue by asako98** [PaddlePaddle/PaddleSeg#1537]
用的自己的数据集，用unet训练
训练出来miou0.4 ,kappa是负值，accuracy倒是很高
请问是数据集数量的问题吗，数据集大概140张
预测也无法分割图像，好像根本没有学习
换了其他模型也是这种结果

**Comment by asako98** [comment]
配置文件
batch_size: 4
iters: 1000

train_dataset:
  type: OpticDiscSeg
  dataset_root: dataset/breast_disc_seg
  transforms:
    - type: Resize
      target_size: [160, 400]
    - type: RandomHorizontalFlip
    - type: Normalize
  mode: train

val_dataset:
  type: OpticDiscSeg
  dataset_root: data/breast_disc_seg
  transforms:
  - type: Resize
    target_size: [160, 400]
  - type: Normalize
  mode: val


optimizer:
  type: sgd
  momentum: 0.9
  weight_decay: 4.0e-5

lr_scheduler:
  type: PolynomialDecay
  learning_rate: 0.01
  end_lr: 0
  power: 0.9

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

**Comment by asako98** [comment]
预测也分割不出来，感觉根本没学习到

**Comment by juncaipeng** [comment]
评测的时候会输出miou和每类的class miou，你贴出来看下。

**Comment by asako98** [comment]
> 
> 
> 评测的时候会输出miou和每类的class miou，你贴出来看下。

2021-11-15 17:39:51 [INFO]      [EVAL] #Images: 76 mIoU: 0.4906 Acc: 0.9812 Kappa: -0.0008
2021-11-15 17:39:51 [INFO]      [EVAL] Class IoU:
[0.9812 0.    ]
2021-11-15 17:39:51 [INFO]      [EVAL] Class Acc:
[0.9816 0.    ]
请问这个是什么意思。。。

**Comment by juncaipeng** [comment]
说明全部预测为第一类了，第二类的iou为0，所以导致kappa不准确。
可以看看数据是不是类别不均衡。
